scripts/teams.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "name": "core",
+    "users": [
+      "RingerJK",
+      "VysotskiVadim",
+      "dzinad"
+    ]
+  },
+  {
+    "name": "ui",
+    "users": [
+      "abhishek1508",
+      "kmadsen",
+      "tomaszrybakiewicz"
+    ]
+  },
+  {
+    "name": "app",
+    "users": [
+      "Zayankovsky",
+      "korshaknn",
+      "Guardiola31337"
+    ]
+  },
+  {
+    "name": "devops",
+    "users": [
+      "SevaZhukov"
+    ]
+  },
+  {
+    "name": "any",
+    "users": [
+      "LukasPaczos"
+    ]
+  }
+]
```

Comment:
I don't think this is where we should maintain the team structure. Any time there is movement it would require a code change to a public repository.

Reply:
I checked github token from actions can have access only to the repo, it can't fetch GitHub teams